src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-security-lake-monitoring-integration.mdx

```diff
@@ -7,37 +7,108 @@ tags:
 metaDescription: "New Relic's Amazon Security Lake integration: what data it reports, and how to enable it."
 ---
 
-[New Relic infrastructure integrations](/docs/infrastructure/integrations-getting-started/getting-started/introduction-infrastructure-integrations) 
-now include an integration for AWS Security Lake, allowing you to send your security log data to New Relic. 
+[New Relic infrastructure integrations](/docs/infrastructure/integrations-getting-started/getting-started/introduction-infrastructure-integrations) include an integration for AWS Security Lake, allowing you to send your security log data to New Relic. 
 
 Collect and send telemetry data to New Relic from [Security Lake](https://aws.amazon.com/security-lake/) using our integration. You can use this integration to monitor your services, query incoming data, and build dashboards to observe everything at a glance.
 
 ## Activate integration [#activate]
 
-To enable this integration, set up S3 log forwarder. We suggest using our application for ease and convenience, but you can also set your own.
-
-## Configuration 
-
-Install our dedicated [Amazon Security Lake log forwarder](/docs/logs/forward-logs/aws-lambda-sending-security-logs-s3/) but instead of Lambda trigger, use AWS SQS service. Set up variables accordingly.
-
-## Logs
-
-To find your logs on New Relic, go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Logs** and set **Attributes** to `product.name`, choose one or few of your interest.
-
-The followings source types of logs are currently supported:
+To enable this integration, set up an S3 log forwarder. We suggest using our application for ease and convenience, but you can also set your own.
+
+## Configuration steps [#steps]
```

Comment:
* This is missing guidance on [a] How would a customer configure logs to be ouput to begin with ?

```
Complete the steps mentioned in Amazon Security Lake [Getting Started Guide]( https://docs.aws.amazon.com/security-lake/latest/userguide/getting-started.html).
```

* Pre-Requisite

```
Complete the pre-requisites mentioned under `data access` subscriber type as mentioned at [Security Lake Subscriber Guide](https://docs.aws.amazon.com/security-lake/latest/userguide/subscriber-data-access.html#prereqs-creating-subscriber)
```

* Guidance on regions.. What is NR recommendation to install this integration per region basis or use rollup region

```
Consolidate data from multiple regions into a single rollup. We recommend consolidating multiple regions so you can avoid repeating the steps in the next section for each region. 

Amazon Security Lake collects security logs and events across AWS Regions for enabled services. AWS stores data in different Amazon S3 buckets per each region. For more details, see [managing multiple regions](https://docs.aws.amazon.com/security-lake/latest/userguide/manage-regions.html).
```

Reply:
Ok, let me review this and update it, I'll make another commit shortly.